infrastructure/src/main.py

```diff
@@ -66,7 +66,6 @@
             Principal=Principal(
                 "AWS",
                 [
-                    "arn:aws:iam::060170161162:role/eimi.okuno@bbc.co.uk",
                     "arn:aws:iam::060170161162:role/tamsin.green@bbc.co.uk",
```

Comment:
Getting an error about a principal - wondering if we need to add Cosmos to this list?? But then, I don't know why we wouldn't have needed this before... but we have it in LPT.

"arn:aws:iam::060170161162:role/cosmos.service.news-labs.live"

Maybe the error in the AWS console is because I am not a principal?

Reply:
<img width="1008" alt="Screenshot 2021-04-28 at 09 08 52" src="https://user-images.githubusercontent.com/22618458/116369558-6d78d080-a801-11eb-90f7-c1d2ca1a7f1b.png">

Reply:
Eimi’s role doesn’t exist anymore,